### ISSUE TITLE
.pyd files from Python

### ISSUE THREAD

**Issue by ManPython** [ytk2128/dll-merger#4]
Did you try maybe a combine .dll and .pyd files as both in .exe?
Want to support us here with Your solution? :  https://github.com/pyinstaller/pyinstaller/issues/8572 ?

**Comment by ytk2128** [comment]
Never tried, but it looks interesting.

There's still no solution for in-memory execution?

**Comment by ManPython** [comment]
Still not.. poor, right? : )

**Comment by ytk2128** [comment]
I've found [PyOxidizer](https://github.com/indygreg/PyOxidizer) that seems to have features you're looking for.

**Comment by ManPython** [comment]
It's very problematic solution - I know this. It require whole Rust, and producing big chains/elements (going in to GB's, eg. 2GB).
Soo.. pyinstaller need management from files to ram only.

**Comment by ytk2128** [comment]
Yeah.. that's probably because it's embedding self-contained Python interpreters into the executable.

It could be a good solution for some cases, but definitely doesn't seem suitable for your needs.

I'll let you know if I find a better solution, or start working on it someday.

**Comment by ytk2128** [comment]
Have you used PyInstaller-like software with the obfuscation feature?

Isn't it relatively safer since the unpacked source codes are obfuscated.

**Comment by ManPython** [comment]
Yes, I tried and have own, but can't be used with success due unpacking - many problems around as issues of this type external tools, not internal like PyInstaller
https://github.com/orgs/pyinstaller/discussions/8456#discussioncomment-9344860

Most of I tried a VM cpu for code as well know fro C# etc. 

It's more comfy with other PyInstaller but other are not comfy with other libs.. soo.. "bad loop".
I did not check a obfuscation for PyOxidizer, coz is relative new.
